examples/multiplatform/xcodeproj_targets.bzl

```diff
@@ -7,6 +7,8 @@ load(
 
 XCODEPROJ_TARGETS = [
     "//examples/multiplatform/iMessageApp",
+    "//examples/multiplatform/tvOSApp/Test/UnitTests:ExampleTests",
+    "//examples/multiplatform/tvOSApp/Test/UITests:ExampleUITests",
```

Comment:
So these are actually sorted by the label name, so can you place these after the `//examples/multiplatform/tvOSApp` entry?